docs/src/main/sphinx/release/release-459.md

```diff
@@ -0,0 +1,85 @@
+# Release 459 (25 Sep 2024)
+
+## General
+
+* Improve error reporting on input validation for the `approx_percentile`
+  function. ({issue}`23546`)
+* Fix possible query failure when `retry_policy` is set to `TASK` and when
+  adaptive join reordering is enabled. ({issue}`23407`)
+
+## Docker image
+
+* Update Java runtime to Java 23. ({issue}`23482`)
+
+## CLI
+
+* Display data sizes and rates with binary (1024-based) abbreviations such as
+  `KiB` and `MiB`. Add flag `--decimal-data-size` to use decimal (1000-based)
+  values and abbreviations such as `KB` and `MB`. (#13054)
+
+## BigQuery connector
+
+* Add support for dereference pushdown. ({issue}`23443`)
```

Comment:
"dereference pushdown" is planner/IR jargon. Rephrase as "Improve performance of queries that access only a subset of fields from nested data".